.circleci/config.yml

```diff
@@ -8,6 +8,60 @@ jobs:
       - SDK_FILE: "openwrt-sdk-ar71xx-generic_gcc-7.3.0_musl.Linux-x86_64.tar.xz"
       - BRANCH: "master"
     steps:
+      - checkout:
+          path: ~/openwrt_packages
+
+      - run:
+          name: Check changes / verify commits
+          working_directory: ~/openwrt_packages
+          command: |
+             cat >> $BASH_ENV <<EOF
+             echo_red()   { printf "\033[1;31m\$*\033[m\n"; }
+             echo_green() { printf "\033[1;32m\$*\033[m\n"; }
+             echo_blue()  { printf "\033[1;34m\$*\033[m\n"; }
+             EOF
+             source $BASH_ENV
+
+             # Figure out commit_range ($COMMIT_RANGE returned)
+             .circleci/get_commit_range.sh
+             source $BASH_ENV
```

Comment:
duplicate ?

Reply:
Not duplicate - the shell script modifies the BASH_ENV file and this line reads the returned value: COMMIT_RANGE.